plugins/kubernetes/src/components/Pods/PodLogs/PodLogs.tsx

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+import { DismissableBanner, LogViewer } from '@backstage/core-components';
+import { Paper } from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+
+import { ContainerLogContext } from './types';
+import { usePodLogs } from './usePodLogs';
+
+interface PodLogsProps {
+  previous?: boolean;
```

Comment:
Let's drop this since it's currently unused; we can explore the use cases later

Reply:
fixed!